jupyter_server_proxy/utils.py

```diff
@@ -49,3 +51,13 @@ def validate(self, obj, value):
             return value
         else:
             self.error(obj, value)
+
+
+def check_pid(pid):
+    """Check For the existence of a unix pid"""
```

Comment:
We're hoping to add Windows support soon to jupyer-server-proxy. Is this unix specific, or should it work on Windows too?

Reply:
This is not needed anymore. We can check this directly on the simpervisor proc object. If simpervisor supports Windows, that should be working here as well!!

Reply:
Seen to be resolved by d1653d11babfd9717a5b81a2e42e3031d3c79462